### PR TITLE
Unit converter

### LINKED ISSUE #130: Unit_Converter
**Describe the project you want to add with tech stack**

Hi Sir,
**This is the project where I have made a basic unit converter(feet-inches) with the help of HTML , CSS & JS. Please assign me this issue**
**Expected behavior**

This is the project where I have made a basic unit converter(feet-inches) with the help of HTML , CSS & JS
**Screenshots (optional)**

**Additional context (optional )**

Please Assign Me this issue.

### PR DESCRIPTION
**Pull request by apu52**
# Title and Issue number 

Title : Unit Converter

Issue No. : #130 

Code Stack : HTML, CSS ,JS

Close #130 

# Checklist:

- [X] I have mentioned the issue number in my Pull Request.
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have created a helpful and easy to understand `README.md`
- [X] I have updated the Index.html file for my contribution

### CODE REVIEW

**Comment by apu52** [comment]
@sadanandpai Please take a review of my PR & please merge it.

File: native/helpers/challenges.js [hunk 1]
```diff
@@ -379,4 +379,18 @@ export const challenges = [
   { title: 'Emoji Editor', link: '#' },
   { title: 'Sort Table', link: '#' },
   { title: 'Infinite Scroll', link: '#' },
+  {
+    title: 'Food Website',
+    link: 'https://65072ad6fbb37b52ebcf0b6c--radiant-macaron-33a884.netlify.app/#',
```
Comment (sadanandpai) [inline native/helpers/challenges.js:384]: The link should be to the challenge in the repo only

File: native/helpers/challenges.js [hunk 2]
```diff
@@ -379,4 +379,18 @@ export const challenges = [
...
+    difficulty: 'moderate',
+    developer: 'apu52',
+    tags: [],
+  },
+  {
+    title: 'Unit Converter',
+    link: 'https://apu52.github.io/Unit-converter/',
```
Comment (sadanandpai) [inline native/helpers/challenges.js:391]: The link should be to the challenge in the repo only

File: native/machine-coding/Unit converter/Contributing.md
Comment (sadanandpai) [inline native/machine-coding/Unit converter/Contributing.md:1]: why do we have this file

File: native/machine-coding/Unit converter/Contributors.md
Comment (sadanandpai) [inline native/machine-coding/Unit converter/Contributors.md:1]: why do we have this file

File: native/machine-coding/Unit converter/README.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# Unit-converter
+
+# Description
+This is my 1st project in my internship of Code Clause where I have made a very simple Stopwatch using i)HTML
+ii)CSS &
+iii)JS
```
Comment (sadanandpai) [inline native/machine-coding/Unit converter/README.md:6]: Please keep a valid description for users to understand about the project

File: native/machine-coding/Unit converter/index.html [hunk 4]
```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Unit Converter</title>
+    <link rel="stylesheet" href="style.css" />
```
Comment (sadanandpai) [inline native/machine-coding/Unit converter/index.html:8]: please include the required tags

File: native/machine-coding/Unit converter/index.js [hunk 5]
```diff
@@ -0,0 +1,19 @@
+var feet = document.getElementById("feet");
+var inch = document.getElementById("inch");
```
Comment (sadanandpai) [inline native/machine-coding/Unit converter/index.js:2]: use const

File: native/machine-coding/Unit converter/style.css [hunk 6]
```diff
@@ -0,0 +1,51 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: "Poppins", sans-serif;
+    text-transform: uppercase;
+  }
```
Comment (sadanandpai) [inline native/machine-coding/Unit converter/style.css:7]: do not modify font family

**Review by sadanandpai — CHANGES_REQUESTED** [review]

**Comment by apu52** [comment]
@sadanandpai I have resolved the conflicts...Can you please take a look & please merge it

**Comment by apu52** [comment]
I have deleted those files & have changed the codes . @sadanandpai Can you please take a review & please merge it

**Comment by sadanandpai** [comment]
@apu52 please address the comments properly. Do not mark resolved without fixing.

**Review by sadanandpai — CHANGES_REQUESTED** [review]

**Pull request closed without merging**